api/factories.py

```diff
@@ -1,14 +1,6 @@
 import factory.fuzzy
-from .models import *
-from faker import Faker
-from faker.providers import date_time
-
 
-class FakeObjects:
-    fake = Faker().add_provider(date_time)
-
-    def get_fake(self):
-        return self.fake
+from .models import *
```

Comment:
Please, do not use this import all, but explicitly import all required models.

https://pythonconquerstheuniverse.wordpress.com/2011/03/28/why-import-star-is-a-bad-idea/

Reply:
done